asm/interpreter.asm

```diff
@@ -79,8 +79,7 @@ quit_reset
     stx     SAVE_INPUT_STACK_DEPTH
 
     jsr     close_all_logical_files
```

Comment:
It looks like close_all_logical_files is only used here. So it could even make sense to inline it.